packages/core/src/ui/widgets/Widget.ts

```diff
@@ -0,0 +1,125 @@
+export type WidgetProps = {
+  id: string;
+  container?: HTMLElement;
+};
+
+type WidgetSize = {
+  width: number;
+  height: number;
+};
+
+class Widget {
```

Comment:
Ok maybe we need this Widget class, I was wondering if we can just use cornerstoneTools.utilities.voi.colorbar 

How about we just expose this widget class in utilities root? 

I really don't want to mention ui or widget in src

Reply:
requires documentation on what is it and how to use it